test/terraform/aws-persistent/main.tf

```diff
@@ -24,6 +24,15 @@ module "materialize_infrastructure" {
   environment  = "dev"
   install_materialize_operator = true
 
+  helm_values = {
+      defaultReplicationFactor = {
+          system = 1
+          probe = 1
+          support = 1
+          analytics = 1
```

Comment:
I think the expected defaults should be 
```
system = 0
probe = 0
support = 0
analytics = 0
```

Reply:
But the tests expect these clusters to have replicas and be responsive, I'm seeing test failures otherwise, and would prefer not to exclude more tests.

Reply:
Ahh that makes sense!